decidim-surveys/db/migrate/20180405015147_create_decidim_survey_answer_choices.rb

```diff
@@ -26,8 +26,9 @@ def up
 
     SurveyAnswer.find_each do |answer|
       question = SurveyQuestion.find_by(id: answer.decidim_survey_question_id)
+      choices = SurveyAnswerChoice.where(decidim_survey_answer_id: answer.id)
 
-      answer.choices.each do |answer_choice|
+      choices.each do |answer_choice|
```

Comment:
Why this change?

Reply:
`answer.choices` didn't work as we're using a fake model. This migration never worked.

Reply:
OMG

Reply:
Shit, I tried these convoluted migrations with real data, but I must have missed something... :(